README.md

```diff
@@ -35,7 +35,7 @@ In order to run the tests, enter:
 
     make check
 
-The tests need Perl to be built and run. If you don't have Perl installed, you can skip building the tests with:
+The tests need Python to be built and Perl to be run. If you don't have one of them installed, you can skip building the tests with:
```

Comment:
Shouldn't this be "If you don't have `any` of them installed"?

Reply:
I don't think so.
It's enough for one of them to not be installed in order not to be able to run the tests. ( Python is used to build them and Perl is used to run them, unfortunately )

Reply:
I think it would be more natural to say "If you don't have one of *these* installed...", but I don't think it's worth changing.